app/Resources/views/base.html.twig

```diff
@@ -4,26 +4,37 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=EDGE">
     <title>Atlassian Vendor Tools</title>
     <link rel="stylesheet" href="{{ asset('assets/aui/css/aui.min.css') }}" media="all">
     <link rel="stylesheet" href="{{ asset('assets/aui/css/aui-experimental.min.css') }}" media="all">
     <link rel="stylesheet" href="{{ asset('assets/css/mpcrm.css') }}" media="all">
-    <link rel="shortcut icon" href="/bundles/app/favicon.ico" type="image/x-icon" />
+    <link rel="shortcut icon" href="/bundles/app/favicon.ico" type="image/x-icon"/>
     <link rel="icon" href="/bundles/app/favicon.ico" type="image/x-icon">
 </head>
 <body class="aui-page-fixed">
 <div id="page">
     <header id="header" role="banner">
-        <nav class="aui-header" role="navigation">
+        <nav class="aui-header" role="navigation" style="min-height: 40px">
```

Comment:
Move inline style to the appropriate file:
https://github.com/thecatontheflat/atlassian-vendor-tools/blob/master/web/assets/css/mpcrm.css